src/main/java/Duke.java

```diff
@@ -45,24 +69,45 @@ public static void main(String[] args) {
 
                 System.out.println("____________________________________________________________\n"
                         + "Well done! I've marked this task as done: \n"
-                        + "["
-                        + taskList[i].getStatusIcon()
-                        + "] "
-                        + taskList[i].description
-                        + "\n"
+                        + taskList[i]
+                        + System.lineSeparator()
                         + "____________________________________________________________\n"
                 );
 
-            } else {
+            } else if (isProperTask) {
+
+                if (isTodoTask) {
+                    taskList[itemCount] = new Todo(line.substring(5));
+                } else if (isDeadlineTask) {
+                    int indexOfSlash = line.indexOf("/");
+                    String actualTask = line.substring(9, indexOfSlash-1);
+                    String deadlineBy = line.substring(indexOfSlash+4);
+                    taskList[itemCount] = new Deadline(actualTask, deadlineBy);
+                } else {
+                    int indexOfSlash = line.indexOf("/");
+                    String actualTask = line.substring(6, indexOfSlash-1);
+                    String eventAt = line.substring(indexOfSlash+4);
+                    taskList[itemCount] = new Event(actualTask, eventAt);
+                }
+
+                itemCount++;
                 System.out.println("____________________________________________________________\n"
-                        + "added: "
-                        + line + "\n"
+                        + "Got it! I've added this task: "
+                        + System.lineSeparator()
+                        + "  "
+                        + taskList[itemCount - 1]
+                        + System.lineSeparator()
+                        + "Now you have "
+                        + Integer.toString(itemCount)
+                        + " tasks left in the list."
+                        + System.lineSeparator()
```

Comment:
For this part, maybe can consolidate all the words you want to print into one line, so it is easier to read also